CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+GOBBLIN 0.9.0
+-------------
+
+### Created Date: 12/13/2016
+
+## Highlights
```

Comment:
It seems mostly irrelevant for users (only developers care). But if you think it's important, I'll include it.